tests/carrier_account_test.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+import easypost
+import tests.fixture as fixture
+
+
+def generate_carrier_account():
+    return {
+        "type": "UpsAccount",
+        "credentials": {
+            "account_number": "A1A1A1",
+            "user_id": "USERID",
+            "password": "PASSWORD",
+            "access_license_number": "ALN",
+        },
+    }
+
+
+@pytest.mark.vcr()
+def test_carrier_account_create(prod_api_key):
```

Comment:
Is prod key actually being used in here? Check for all these tests

Reply:
Yes, the default API key is using the test key, we need to pass in prod_api_key for carrier account testing since it require prod key.

Reply:
What I'm saying is, it doesn't seem like the prod key is actually getting passed along to the inner functions to actually be used? Or am I missing some syntactical sugar?

Reply:
`prod_api_key` is coming from `conftest.py` as a fixture. Check out this doc: https://docs.pytest.org/en/latest/how-to/fixtures.html